README.md

```diff
@@ -103,11 +111,14 @@ You must have the following components available in order to run Firecracker
 microVMs with containerd:
 
 * containerd >= 1.2
-* Firecracker >= 0.10.1 with [vsock support](https://github.com/firecracker-microvm/firecracker/blob/master/docs/experimental-vsock.md) enabled.
+* Firecracker >= 0.10.1 with [vsock
```

Comment:
We should probably update that to something more recent, like 0.15.x. I'm not sure that the API has changed in incompatible ways, but we're not actively working to maintain compatibility with older releases.

Reply:
Done